packages/labs/ssr/src/test/lib/render-lit_test.ts

```diff
@@ -218,6 +218,46 @@ test('element with property', async () => {
   );
 });
 
+test('element with attribute', async () => {
```

Comment:
These tests should be added here: https://github.com/lit/lit/blob/main/packages/labs/ssr/src/test/integration/tests/basic.ts#L883

This way we get server rendering with an assertion, then client hydration that checks that there's no mutations.

Reply:
for "normal" rendering it's already there [AttributePart accepts undefined](https://github.com/lit/lit/blob/main/packages/labs/ssr/src/test/integration/tests/basic.ts#L940) and that worked before already

it's about the rendering of attributes of an element... so I added it to `LitElement: Attribute binding`